app/indexers/identifiable_indexer.rb

```diff
@@ -3,6 +3,8 @@
 class IdentifiableIndexer
   attr_reader :cocina
 
+  DEPRECATED_CATALOGS = %w[symphony previous].freeze
```

Comment:
Along with my suggestion below:

```suggestion
  CURRENT_CATALOG_TYPE = 'folio'
```